### ISSUE TITLE
Feedback: com.gradle.plugin-publish release notes are confusing

### ISSUE THREAD

**Issue by TWiStErRob** [gradle/gradle#26306]
### Issue type

Wrong or misleading information

### Problem description

TL;DR: compare these two release notes
 * https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.0
 * https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.1

The latter duplicates points from the first, which could cause extra work for readers, a story of such below.

### Context (optional)

_I'm reviewing Renovate PRs and looking at release notes of the changes included in the update._

While reviewing [this PR](https://github.com/gradle-nexus/publish-plugin/pull/262/files) I opened [1.2.1 release notes](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.1) to see what changed:
> Version 1.2.1 Created 17 August 2023.
>
> - Add `--validate-only` option to `publishPlugins` task to get feedback from server-side plugin validation without actually publishing.
> - Add support for publishing `*.sigstore` bundle files as an alternate signing solution.
> - Fix integration with Shadow plugin that causes JAR manifest to contain Gradle API jars.
> - Adjust logging levels from LIFECYCLE to INFO.

The 3rd one looked familiar as I remembered I reported something related to "Shadow JAR" found in our repo, and that it was resolved quickly. I looked up the [issue](https://github.com/gradle/gradle/issues/23928) and found [the PR that removed the workaround](https://github.com/gradle-nexus/publish-plugin/pull/212) listed in the references at the bottom. It was the PR that updated 1.1.0 to 1.2.0. This is where I got confused: did Gradle fix the same thing again? Was it not merged to the right branch in our repo? And other doubting questions. Then I opened [1.2.0 release notes](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.0):
> Version 1.2.0 Created 07 April 2023.
> - Add `--validate-only` option to `publishPlugins` task to get feedback from server-side plugin validation without actually publishing.
> - Add support for publishing `*.sigstore` bundle files as an alternate signing solution.
> - Fix integration with Shadow plugin that causes JAR manifest to contain Gradle API jars.

and noticed that 3 out of the 4 bullet points in the new version were already listed in the previous version. So there are duplicates in the release notes and the actual changes are:

> Version 1.2.0 Created 07 April 2023.
> - Add `--validate-only` option to `publishPlugins` task to get feedback from server-side plugin validation without actually publishing.
> - Add support for publishing `*.sigstore` bundle files as an alternate signing solution.
> - Fix integration with Shadow plugin that causes JAR manifest to contain Gradle API jars.
> 
> Version 1.2.1 Created 17 August 2023.
> - Adjust logging levels from LIFECYCLE to INFO.

Note: another Gradle plugin has the release note lines attributed the right versions, compare these two:
 * https://plugins.gradle.org/plugin/com.gradle.enterprise/3.14
 * https://plugins.gradle.org/plugin/com.gradle.enterprise/3.14.1

### Page with the problem

https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.1

**Comment by ljacomet** [comment]
Thank you for your interest in Gradle!

This is a valid documentation issue that we will address.

***

Only that added line should appear in the latest version

**Comment by ryandens** [comment]
:wave: Hi folks, I noticed this happened again with the 1.2.2 release. Understood it's human error and likely no code changes to make here in the short term, but just wanted to let you know

```
1.2.2

Created 29 August 2024.
- Add `--validate-only` option to `publishPlugins` task to get feedback from server-side plugin validation without actually publishing.
- Add support for publishing `*.sigstore` bundle files as an alternate signing solution.
- Fix integration with Shadow plugin that causes JAR manifest to contain Gradle API jars.
- Adjust logging levels from LIFECYCLE to INFO.
- Add proper annotations to tasks so validatePlugins passes in the strict mode
```


```
Version 1.2.1

1.2.1

Created 17 August 2023.
- Add `--validate-only` option to `publishPlugins` task to get feedback from server-side plugin validation without actually publishing.
- Add support for publishing `*.sigstore` bundle files as an alternate signing solution.
- Fix integration with Shadow plugin that causes JAR manifest to contain Gradle API jars.
- Adjust logging levels from LIFECYCLE to INFO.
```

**Comment by ljacomet** [comment]
Hey Ryan,

My mistake, I thought it made sense to do it that way for patch releases and did not remember this explicit request.

Description has been updated to focus on the real changes and not accumulate them.

**Comment by ryandens** [comment]
Thank you!
